openbb_platform/obbject_extensions/charting/openbb_charting/core/backend.py

```diff
@@ -16,9 +14,9 @@
 
 try:
     from pywry import PyWry  # pylint: disable=import-outside-toplevel
-except ImportError as e:
-    if Env().DEBUG_MODE:
-        print(f"\033[91m{e}\033[0m")  # noqa: T201
+except ImportError:
+    # if Env().DEBUG_MODE:
```

Comment:
Can we clean this up please?

Reply:
Do you mean, remove the commented lines? If so, done.